Share/Share.Plugin.iOSUnified/ShareImplementation.cs

```diff
@@ -157,19 +150,28 @@ public Task<bool> ShareLink(string url, string message = null, string title = nu
             shareMessage.Text = message;
             shareMessage.Url = url;
 
-            var shareOptions = new ShareOptions();
-            shareOptions.ExcludedUIActivityTypes = excludedActivityTypes?.Select(x => (string)x).ToArray();
+            return Share(shareMessage, null, excludedActivityTypes);
+        }
 
-            return Share(shareMessage, shareOptions);
+        /// <summary>
+        /// Share a message with compatible services
+        /// </summary>
+        /// <param name="message">Message to share</param>
+        /// <param name="options">Platform specific options</param>
+        /// <returns>True if the operation was successful, false otherwise</returns>
+        public Task<bool> Share(ShareMessage message, ShareOptions options = null)
+        {
+            return Share(message, options, null);
         }
 
         /// <summary>
         /// Share a message with compatible services
         /// </summary>
         /// <param name="message">Message to share</param>
         /// <param name="options">Platform specific options</param>
+        /// <param name="excludedActivityTypes">UIActivityTypes that should not be displayed</param>
         /// <returns>True if the operation was successful, false otherwise</returns>
-        public async Task<bool> Share(ShareMessage message, ShareOptions options = null)
+        private async Task<bool> Share(ShareMessage message, ShareOptions options = null, params NSString[] excludedActivityTypes)
```

Comment:
Should we make this public? I believe this would deter the dev from using the new ShareOptions api. Calling this method is a bit cumbersome anyway, since you would have to cast CrossShare.Current to ShareImplementation first.

I would like to remove this method in the future once we've phased out the old Share and ShareLink methods.